l10n_be_cooperator/__manifest__.py

```diff
@@ -13,7 +13,6 @@
         "cooperator",
         "cooperator_website",
         "l10n_be",
-        "l10n_be_invoice_bba",
```

Comment:
Note : I removed it in 14 because the module was empty. In 12 it is not empty. In any case, since it is on auto-install it will be installed anyway. 
I think that maybe we should keep it, since we depend on it for invoicing and we removed the methods ` def get_representative_valst` to replace it with this module.

Reply:
i think that this dependency is a leftover of version 9, where the payment reference type preference was per partner instead of company-wide. i don’t find evidence that `l10n_be_cooperator` needs `l10n_be_invoice_bba` to work properly. also, since it’s autoinstallable, leaving this dependency means that the module would be installed automatically only if `l10n_be_invoice_bba` was installed also (but since the latter is also autoinstallable on `account` and `l10n_be`, it doesn’t matter).

note that `get_representative_valst()` never worked since there was a trailing `t` in its name compared to the original method. 🤦

Reply:
Lol, OK thanks for the explanation :)